pages/auth/authorize.vue

```diff
@@ -68,6 +86,32 @@ import { useAuth } from '@/composables/auth.js'
 
 import { useScopes } from '@/composables/auth/scopes.ts'
 
+const { formatMessage } = useVIntl()
+
+const messages = defineMessages({
+  appInfo: {
+    id: 'auth.authorize.app-info',
+    defaultMessage:
+      '<strong>{appName}</strong> by <creator-link>{creatorUsername}</creator-link> will be able to:',
+  },
+  authorizeAppName: {
+    id: 'auth.authorize.authorize-app-name',
+    defaultMessage: 'Authorize {app_name}',
+  },
```

Comment:
```suggestion
  title: {
    id: 'auth.authorize.title',
    defaultMessage: 'Authorize {appName}',
  },
```
This appears to be a page title, so it can be keyed like one. Also, camelCase for vars :)